src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1389,6 +1394,47 @@
             }
         },
 
+        "reportingDescriptorTranslation": {
+            "description": "tbd",
+            "additionalProperties": false,
+            "type": "object",
+            "properties": {
+
+                "id": {
+                    "description": "TBD matches reportingDescriptor.id",
```

Comment:
>TBD matches reportingDescriptor.id [](start = 36, length = 34)

"The stable, opaque identifier of the reporting descriptor to which this translation belongs, matching reportingDescriptor.id."

Reply:
Please do a scrub to ensure that ": matches xxx" becomes ", matching xxx" throughout. (note punctuation)

---
In reply to: [263094418](https://github.com/Microsoft/sarif-sdk/pull/1336#discussion_r263094418) [](ancestors = 263094418)